pkg/monitors/mongodbatlas/utils.go

```diff
@@ -0,0 +1,169 @@
+package mongodbatlas
+
+import (
+	"github.com/Sectorbob/mlab-ns2/gae/ns/digest"
+	"github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"github.com/signalfx/golib/v3/datapoint"
+	"github.com/sirupsen/logrus"
+)
+
+func newClient(publicKey, privateKey string) (*mongodbatlas.Client, error) {
+	//Setup a transport to handle digest
+	transport := digest.NewTransport(publicKey, privateKey)
+
+	//Initialize the client
+	client, err := transport.Client()
+	if err != nil {
+		return nil, err
+	}
+
+	//Initialize the MongoDB Atlas API Client.
+	return mongodbatlas.NewClient(client), nil
+}
+
+func newFloatValue(dataPoints []*mongodbatlas.DataPoints) datapoint.FloatValue {
+	if len(dataPoints) == 0 || dataPoints[0].Value == nil {
+		return nil
+	}
+	return datapoint.NewFloatValue(float64(*dataPoints[0].Value))
+}
+
+func newIntValue(dataPoints []*mongodbatlas.DataPoints) datapoint.IntValue {
+	if len(dataPoints) == 0 || dataPoints[0].Value == nil {
+		return nil
+	}
+	return datapoint.NewIntValue(int64(*dataPoints[0].Value))
+}
+
+func logErrors(logger *logrus.Logger, errs ...error) bool {
+	var logged bool
+	if errs != nil {
+		for _, err := range errs {
+			if err != nil {
+				logger.Error(err)
```

Comment:
To log an error use `logger.WithError(err).Error("Atlas responded with an error")` or something like that.